index.js

```diff
@@ -8,24 +8,23 @@ var TrackerGroup = exports.TrackerGroup = function (name) {
   EventEmitter.call(this)
   this.name = name
   this.trackGroup = []
-  var self = this
   this.totalWeight = 0
-  var noteChange = this.noteChange = function (name) {
-    self.emit("change", name || this.name)
-  }.bind(this)
-  this.trackGroup.forEach(function(unit) {
-    unit.on("change", noteChange)
-  })
+  this.noteChange = TrackerGroup.prototype.noteChange.bind(this);
```

Comment:
This is actually slower than changing the above `this.name` to `self.name` and ditching the bind, because bind is terrifyingly slow.